TutorialApplication/Organization.h

```diff
@@ -7,28 +7,17 @@
 //
 
 #import <Foundation/Foundation.h>
+#import "FFOrganization+CoreDataClass.h"
+#import "FFOrganization+CoreDataProperties.h"
+@class FFEmployee;
 
-@class Employee;
+@interface FFOrganization (Organization)
 
-@interface Organization : NSObject
+- (FFEmployee*)addEmployeeWithName:(NSString *)nameEmployee;
 
-@property (strong, nonatomic, readonly) NSArray<Employee *> *employees;
+- (NSString *)employeeWithLowestSalary:(NSArray*)myEmployees;
 
-@property (strong, nonatomic) NSString *name;
-
-- (id)initWithName:(NSString *)name;
-
-- (void)addEmployeeWithName:(NSString *)nameEmployee;
-
-- (void)addEmployee:(Employee *)employee;
-
-- (void)removeEmployee:(Employee *)employee;
-
-- (int)calculateAverageSalary;
-
-- (NSString *)employeeWithLowestSalary;
-
-- (NSArray *)employeesWithSalary:(int)salary tolerance:(int)tolerance;
+- (NSArray *)employeesWithSalary:(int)salary tolerance:(int)tolerance employess:(NSArray*)myEmployees;
```

Comment:
You shouldn't pass employees array here, since organization class already knows about its employees.